### PR TITLE
Add my profile

### PR DESCRIPTION
**Pull request by rhen99**
I just added my profile, rhen99.json

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1608"></a>

### CODE REVIEW

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,
Revert the `package-lock.json` file changes. If you have issues, feel free to reach us out.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by rhen99** [comment]
I didn't change anything in the `package.lock.json`. I just forgot to commit it. That's all.

File: package-lock.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.17",
```
Comment (kumarsonsoff3) [inline package-lock.json:9]: Revert the package version back to the one in base branch! 
```suggestion
      "version": "0.70.1",
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]
Yea @rhen99, you didn't made any changes there, but since you run `npm` when you start, it automatically updates the package version to the latest, so it is! 

You should have only commit the file you're changing, but no worries! ~~
<hr/>

Just make these 2 changes now, it will then good to go! 
- Change the package version back to the initial (line no `9`) as suggested below!
- Remove the `empty endline` from the `package-lock.json` file! 

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Comment by EmmaDawsonDev** [comment]
Did you need any help @rhen99 ?

**Comment by EmmaDawsonDev** [comment]
Since there's been no progress on this for almost 3 weeks I'll close it. Please feel free to open a new pull request with your profile but not including the package-lock file.

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Pull request closed without merging**